sympy/solvers/tests/test_solveset.py

```diff
@@ -2895,3 +2895,33 @@ def test_issue_21236():
 def test_issue_21908():
     assert nonlinsolve([(x**2 + 2*x - y**2)*exp(x), -2*y*exp(x)], x, y
                       ) == {(-2, 0), (0, 0)}
+
+
+def test_issue_19144():
```

Comment:
This got included somehow without the changes of the PR that introduced it in #21989

Reply:
I must have messed up the conflict when I cherrypicked.